### PR TITLE
Fix NPE while adding "response_body_size" breadcrumb (#1907)

### PR DESCRIPTION
**Pull request by neseleznev**
## :scroll: Description

Closes https://github.com/getsentry/sentry-java/issues/1907 , see description there

## :green_heart: How did you test it?

Tested by researching API of dependency (OpenFeign)

## :pencil: Checklist
- [x] I reviewed the submitted code
- [ ] I didn't add tests, because it requires significant refactoring so that length could be `null` Integer
- [x] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Breaking changes:
 `Timber.tag` is no longer supported by our [Timber integration](https://docs.sentry.io/platforms/android/configuration/integrations/timber/) and will not appear on Sentry for error events. 
 Please vote on this [issue](https://github.com/getsentry/sentry-java/issues/1900), if you'd like us to provide support for that.
 
+* Fix: NPE while adding "response_body_size" breadcrumb, when response body length is unknown (#1907)
```
Comment (maciejwalkowiak) [inline CHANGELOG.md:11]: Please move this line to line 6, above breaking changes.

**Review by maciejwalkowiak — APPROVED** [review]
Thanks for fix!

File: CHANGELOG.md (hunk 1 above)
Comment (neseleznev) [reply CHANGELOG.md]: @maciejwalkowiak All set

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 * Fix: Do not include stacktrace frames into Timber message (#1898)
+* Fix: NPE while adding "response_body_size" breadcrumb, when response body length is unknown (#1907)
```
Comment (marandaneto) [inline CHANGELOG.md:6]:
```suggestion
* Fix: NPE while adding "response_body_size" breadcrumb, when response body length is unknown (#1908)
```

**Comment by romtsn** [comment]
Thanks @neseleznev for fixing this!
